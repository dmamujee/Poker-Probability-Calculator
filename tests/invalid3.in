3 H
3 H
3 S
3 C
3 H
3 H
3 H