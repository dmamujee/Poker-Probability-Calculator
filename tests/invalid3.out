
ERROR: game.cc: Given hand is same card as communal card
Card0: 3 of Hearts
Card1: 3 of Hearts
Hand Ranking: Invalid
